.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file should be in `.gitignore`, could you remove it and add `.vscode` to the ignore list?

Reply:
why ? isn't this file useful, to allow debugging the brokers ?

Reply:
It's `.vscode` specific though. So far we've preferred editor-agnostic utils for testing, such as the [`Makefile`](https://github.com/eclipse/che-plugin-broker/blob/master/Makefile)

Reply:
I added `.vscode/` in `.gitignore` and remove the `launch.json` vscode-specific file.